### PR TITLE
init

### PR DESCRIPTION
**Pull request by nataliapanferova**

### CODE REVIEW

**Comment by nataliapanferova** [comment]
didn't manage to get the syntaxe highlighting to work

**Comment by mittsh** [comment]
@nataliapanferova that's probably because we didn't include C Sharp grammar for syntax highlighting in Paw, I'll do that in 2.3.4

**Comment by nataliapanferova** [comment]
@mittsh ok, thanks, could you please review it?

File: CSCodeGenerator.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+		return env.renderString(frame, {requests: rendered_requests});
+
+	
+	}
+		
+
+}
+
+CSCodeGenerator.identifier = "com.luckymarmot.CSCodeGenerator";
```
Comment (mittsh) [inline CSCodeGenerator.js]: Use `com.luckymarmot.PawExtensions.CSharpCodeGenerator` instead (add `.PawExtensions` and write fully `CSharp`)

**Comment by mittsh** [comment]
@nataliapanferova here's how it looks with highlighting 😄 

image

**Comment by mittsh** [comment]
A few things:
- there are too many spaces between statements, so probably we should make them more regular (an idea: one line skipped when doing something very different, no line skipped otherwise…)
- add a newline at the end of the generated code (UNIX standard)

Probably for later:
- URL parameters would be nicer if generated outside of the URL string, and appended one by one
- same for JSON and Form URL-Encoded bodies

**Comment by mittsh** [comment]
@nataliapanferova for the highlighting, it will be shipped with Paw 2.3.4

**Comment by nataliapanferova** [comment]
@mittsh fixed lines and reference, we can do the other improvements in the next version?

**Comment by mittsh** [comment]
@nataliapanferova Thanks! Yeah sure...merging!

**Comment by mittsh** [comment]
👍

**Pull request merged**